id-repository/id-repository-identity-service/src/main/java/io/mosip/idrepository/identity/service/impl/IdRepoServiceImpl.java

```diff
@@ -691,6 +690,27 @@ private String updateXML(byte[] inputBioData, byte[] existingBioData,AnonymousPr
 		anonymousProfileDto.setNewCbeff(encodeUpdateCbeff);
 		return encodeUpdateCbeff;
 	}
+	
+	private Map<String, BIR> getBIRByType(List<BIR> inputBIRData) {
```

Comment:
can we rename this to getBIRDataMapByType ?
Also can we rename inputBIRData to just cbeffBIRData so it becomes common since we call the same for input and existing ?

Reply:
Done